phpBB/viewforum.php

```diff
@@ -885,6 +885,26 @@
 
 		$pagination->generate_template_pagination($view_topic_url, 'topicrow.pagination', 'start', $replies + 1, $config['posts_per_page'], 1, true, true);
 
+		/**
+		* Event after the topic data has been assigned to the template
+		*
+		* @event core.viewforum_topic_row_after
+		* @var	array	row			Array with the topic data
+		* @var	array	rowset		Array with topics data (in topic_id => topic_data format)
+		* @var	int		topic_id	The topic ID
+		* @var	array	topic_list	Array with current viewforum page topic ids
+		* @var	array	topic_row	Template array with topic data
+		* @since 3.1.3-RC1
+		*/
+		$vars = array(
+			'row',
+			'rowset',
+			'topic_id',
+			'topic_list',
+			'topic_row',
+		);
+		extract($phpbb_dispatcher->trigger_event('core.viewforum_topic_row_after', compact($vars)));
```

Comment:
move below the assignment of `s_type_switch` and add that to the variable list??